numpy/_core/src/multiarray/usertypes.c

```diff
@@ -618,8 +618,8 @@ legacy_userdtype_common_dtype_function(
  * used for legacy user-dtypes, but for example numeric to/from datetime
  * casts were only defined that way as well.
  *
- * @param from
- * @param to
+ * @param from Source type
+ * @param to Destination type
```

Comment:
```suggestion
 * @param from Source DType
 * @param to Destination DType
```
Let's use DType/descriptor to be precise were possible.